### PR TITLE
Updated gazebo9 build tutorial on Windows

### PR DESCRIPTION
**Pull request by peci1**
This is a port of https://bitbucket.org/osrf/gazebo_tutorials/pull-requests/555/updated-gazebo9-build-tutorial-on-windows to github. The PR doesn't merge now, but @azeey was assigned to look at it on bitbucket and distill the valuable changes to the updated tutorial.

These are the things I needed to do to build gazebo9 on windows 10 from scratch. The old instructions were fairly outdated (even though saying they were tested).

The process tries to replicate instructions in https://bitbucket.org/osrf/release-tools/src/default/jenkins-scripts/lib/generic-default-devel-windows.bat where possible.

I completely deleted the section on building OGRE samples, as it was outdated… and, why should it be here anyways?

Louise Poubel 13 Apr 2020:

> @j-rivero  and  @azeey will look into this in conjunction with pull request #566 and pull request #567.

### CODE REVIEW

File: install_on_windows/tutorial_9-0.md [hunk 1]
```diff
@@ -148,7 +150,7 @@ Windows `cmd` for configuring and building.  You might also need to
         %NMAKE% install
         set CMAKE_PREFIX_PATH=%CMAKE_PREFIX_PATH%;%cd%\install\Release\lib\cmake\ignition-transport4
 
-    You should now have an installation of Ignition Transport in gz-ws/ign-transport/build/install/Release
+    You should now have an installation of Ignition Trasport in gz-ws/ign-transport/build/install/Release
```
Comment (peci1) [inline install_on_windows/tutorial_9-0.md:153]: @j-rivero Why this?

File: install_on_windows/tutorial_9-0.md [hunk 2]
```diff
@@ -74,14 +74,16 @@ Windows `cmd` for configuring and building.  You might also need to
 
     > [Ruby](http://rubyinstaller.org/downloads/)
 
-1. Clone Ignition CMake, Math, Msgs, Transport, Sdformat, and Gazebo:
+1. Clone Ignition CMake, Common, Fuel Tools, Math, Transport, Sdformat, and Gazebo:
```
Comment (peci1) [inline install_on_windows/tutorial_9-0.md]: Msgs are missing from the enumeration.

**Comment by peci1** [comment]
I think I got rid of the configure scripts mainly because I'd need to issue much more PRs to get it fixed :-D And, more importantly, they do not prepare a user build environment in case you want to build more packages in the same console. The configure scripts rather directly set cmake arguments instead of setting reusable environment variables.

How do the proposed changes prevent you from PATH modifications? I think changing a command from a tutorial is much more user-friendly than rewriting a git-tracked file...

**Comment by j-rivero** [comment]
After revisiting the problem recently, our plan is not to fix the current instructions since they are pretty hard for users and maintainers. The plan is to look into new approaches for Gazebo on Windows, @peci1 it would be great if you have time to comment on the options available at https://github.com/osrf/gazebo/issues/2901. Thanks, sorry for the inconvenience.

**Pull request closed without merging**